docs/source/markdown/podman-compose.1.md.in

```diff
@@ -13,6 +13,8 @@ The default compose providers are `docker-compose` and `podman-compose`.  If ins
 
 If you want to change the default behavior or have a custom installation path for your provider of choice, please change the `compose_provider` field in `containers.conf(5)`.  You may also set the `PODMAN_COMPOSE_PROVIDER` environment variable.
 
+By default, `podman-compose` will emit a warning when using an external compose provider (such as `docker-compose`). This warning can be disabled by setting `compose_warning_logs` to false in `containers.conf(5)` or setting the `PODMAN_COMPOSE_WARNING_LOGS` environment variable to false. See the man page for `containers.conf(5)` for more information.
```

Comment:
I believe this should be `podman compose`  podman-compose and docker-compose are both two external commands.

Reply:
Example given in #22904 is /usr/local/bin/docker-compose which is why I used that example. Having podman-compose being referred to as "External compose provider" when invoking podman compose seems ... Confusing, at best, to me - but I'm open to change :) Could it be more vendor neutral? Perhaps something like
"By default, `podman compose` will emit a warning when using an external compose provider. [...]"?